package.json

```diff
@@ -67,7 +67,7 @@
     "d3": "3.5.5",
     "datatables.net": "1.10.10",
     "datatables.net-responsive": "2.0.1",
-    "fec-style": "~1.10",
+    "fec-style": "git+https://github.com/18f/fec-style#site-menu",
```

Comment:
Should we leave this as is, and then push a separate change to bump `fec-style`? Would rather not have this pointing to a branch.